gateway.tf

```diff
@@ -828,3 +1047,15 @@ resource "aws_api_gateway_authorizer" "main" {
   authorizer_uri         = coalesce(join("", aws_lambda_alias.authorizer_live.*.invoke_arn), aws_lambda_function.authorizer.invoke_arn)
   authorizer_credentials = aws_iam_role.authorizer.arn
 }
+
+# #########################################
+# Default Responses
+# #########################################
+resource "aws_api_gateway_gateway_response" "test" {
+  rest_api_id   = aws_api_gateway_rest_api.main.id
+  response_type = "DEFAULT_4XX"
+
+  response_parameters = {
+    "gatewayresponse.header.access-control-allow-origin"  = "'*'"
```

Comment:
we should try to remove the `*` in favour of a more restrictive policy.

Reply:
This is only for default 4XX errors, I don't recall why we put that here but the `"test"` resource name suggests me we could remove entirely this resource maybe?